src/daemon/rpmostreed-deployment-utils.c

```diff
@@ -229,7 +247,11 @@ rpmostreed_deployment_generate_variant (OstreeSysroot *sysroot,
       variant_add_commit_details (&dict, "base-", base_commit);
     }
   else
-    base_checksum = g_strdup (csum);
+    {
+      base_checksum = g_strdup (csum);
+      /* we would not consider endoflife attribute for layered commits */
```

Comment:
I think we still want layered commits to "inherit" the end-of-life aspect though, right? I.e. when I do `rpm-ostree status`, I still want to see the end-of-life message even if my commit is a layered one, right?

So really, I think we should just always call `variant_add_metadata_attribute()` with `base_commit` after this if-statement.